mantidimaging/gui/windows/operations/filter_previews.py

```diff
@@ -167,21 +166,15 @@ def add_difference_overlay(self, diff, nan_change):
         lut = map.getLookupTable(0, 1, 2)
         self.image_diff_overlay.setLookupTable(lut)
 
-    def add_negative_overlay(self, after):
-        after[after > 0] = 0.0
-        after[after < 0] = 1.0
-        pos = np.array([0, 1])
-        color = np.array([[0, 0, 0, 0], OVERLAY_COLOUR_NEGATIVE], dtype=np.ubyte)
-        map = ColorMap(pos, color)
-        self.negative_values_overlay.setOpacity(1)
-        self.negative_values_overlay.setImage(after)
-        lut = map.getLookupTable(0, 1, 2)
-        self.negative_values_overlay.setLookupTable(lut)
+    def add_negative_overlay(self):
+        self.imageview_after.enable_nonpositive_check()
 
     def hide_difference_overlay(self):
         self.image_diff_overlay.setOpacity(0)
 
     def hide_negative_overlay(self):
+        self.imageview_after.enable_nonpositive_check(False)
+        return
         self.negative_values_overlay.setOpacity(0)
```

Comment:
Should this line be here?

Reply:
No, there are a few left over bits like this that I am cleaning up in #1188 .